pmix_governance.tex

```diff
@@ -425,6 +425,30 @@ \subsubsection{ASC Secretaries}\label{asc-secretaries}}
 Alternate for a Member may vote on behalf of the Member subject to the
 one vote per Member limit.
 
+\hypertarget{asc-guidance}{%
+\subsection{ASC Guidance}\label{asc-guidance}}
+
+This section provides general guidance for the PMIx ASC and participants.
+The rationale behind providing this guidance is to outline
+philosophy and best practices the ASC should follow in 
+maintaining the PMIx Standard. 
+
+\hypertarget{release-schedule-guidance}{%
+\subsubsection{Release Schedule Guidance}\label{release-schedule-guidance}}
+
+As described in Section~\ref{release-managers},
+the schedule of major releases of the PMIx
+Standard is determined by the Release Managers in coordination with other
+members of the ASC. There is no prescribed major release schedule that the
+Release Managers must follow. However, the Release Managers will prioritize
+timely releases of substantive changes to the PMIx Standard. Substantive changes
+include: new APIs, keys, or other text in the standard; deprecated or removed

```

Comment:
Thinking out loud: IIUC new `provisional` APIs, key, and attributes are not affected or blocked by major release schedules.  I believe it is only APIs that are being elevated to `stable` or being deprecated that are affected. So should a "substantive change" exclude new `provisional` items?

Reply:
Since major releases are not triggered by changes to provisional APIs, then perhaps it is implied? However, it is likely better to make it explicit if that is what we mean. Suggest:
"However, the Release Managers will prioritize timely releases of substantive changes to the Stable entries in the PMIx Standard"  (adding "to the Stable entries")

Reply:
I think that sounds good (adding "to the Stable entries"). I would also suggest rewording "new APIs, keys, and other text in the standard" to "modifications to APIs, keys, and other text in the standard". Since a "new API" comes in as `provisional` so should be excluded.

Reply:
We'll remove the phrase "new APIs, keys, or other text in the standard; "